cyral/resource_cyral_integration_idp_saml.go

```diff
@@ -67,6 +67,18 @@ func CreateGenericSAMLConfig() ResourceOperationConfig {
 	}
 }
 
+func ValidateGenericSAMLConfig() ResourceOperationConfig {
```

Comment:
The name Validate seems a bit unusual to me. From what I understand this is creating a configuration resource  for the identity provider- please correct me if Im wrong - so I would suggest to name it something like `CreateIdPConfig`